libs/command_line_handling/src/command_line_handling.cpp

```diff
@@ -790,7 +792,11 @@ namespace hpx { namespace util {
         }
 #endif
 
+#ifdef HPX_HAVE_LIB_MPI_BASE
         bool have_mpi = util::mpi_environment::check_mpi_environment(rtcfg_);
+#else
+        bool have_mpi = false;
+#endif
```

Comment:
Is `mpi_base` disabled but the MPI parcelport on a valid configuration which should also call `check_mpi_environment`?

Reply:
I changed the `#ifdef`. @biddisco please check if this makes sense to you.

Reply:
This should also check whether the MPI parcelport is enabled. I even think that this should always be checked as it will return `false` if MPI is not available anyways.

Reply:
Good point. I removed the ifdefs completely from here now, as they just duplicated ifdefs that were inside `check_mpi_environment`.